src/Utility/Reporter.php

```diff
@@ -17,7 +17,7 @@ class Reporter
     public static function report(array $stats)
     {
         $pid = null;
-        if (extension_loaded('pcntl')) {
```

Comment:
Maybe it's better to use `function_exists`? If it available?

Reply:
`function_exists` also works. I feel it is less expressive, but nevertheless changed it.